engine/src/main/java/org/terasology/rendering/nui/animation/SwipeMenuAnimationSystem.java

```diff
@@ -181,10 +181,14 @@ public void update(float delta) {
             // avoid skipping over fast animations on heavy load
             animDelta = 0.1f;
         }
+        if(animDelta == 0.0f) {
+            // in case time is paused
+            animDelta = 0.035f;
```

Comment:
It helps to avoid "magic numbers" and instead make a constant at the class level with a descriptive name, like `private static float PAUSED_ANIMATION_FRAME_TICK_TIME` or something